src/lang-promql/complete/hybrid.ts

```diff
@@ -319,7 +336,11 @@ export function analyzeCompletion(state: EditorState, node: SyntaxNode): Context
       result.push({ kind: ContextKind.Duration });
       break;
     case FunctionCallBody:
-      result.push({ kind: ContextKind.MetricName }, { kind: ContextKind.Function }, { kind: ContextKind.Aggregation });
+      result.push(
+        { kind: ContextKind.MetricName, metricName: getMetricIdentifierValue(node, state) },
```

Comment:
I think this case is not really covered by the unit test. Which is a bit sad, since now I don't remember in what case it can happen :(

But here that's not possible that the node `MetricIdentifier` is a parent of the `FunctionCallBody`. 

The grammar is available [here](https://github.com/promlabs/lezer-promql/blob/master/src/promql.grammar)

Reply:
I even wondering what would happen if we remove completely this case. Since the behavior of the lezer-tree changed since the last version, it's possible that it won't be possible to reach this case.

Reply:
This case is possible and it is covered by the tests 'metric/function/aggregation autocompletion' and 'metric/function/aggregation autocompletion 3' (I realized this when I removed `metricName: getMetricIdentifierValue(node, state)` from the line above and the tests began to fail because the test expected `metricName` to be the empty string).

Reply:
ah cool. Thanks for the test!